sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache.cpp

```diff
@@ -8,6 +8,8 @@
 // UNSUPPORTED: target-nvidia, target-amd
 // UNSUPPORTED-INTENDED: aspect-ext_intel_matrix isn't currently supported for
 // other triples
```

Comment:
Not related to the fix, but based on the comment `REQUIRES:` should work better than `UNSUPPORTED:`.